app/client/ui/OpenAccessibilityModal.ts

```diff
@@ -0,0 +1,184 @@
+        urlState().setLinkUrl({account: 'account'}),
+        t("profile settings"),
+      )
+    }))
+  );
+};
+
+const keyboardSection = () => {
+  const nextRegionShortcut = dom('span', getCssKeys(allCommands.nextRegion.humanKeys));
+  const prevRegionShortcut = dom('span', getCssKeys(allCommands.prevRegion.humanKeys));
+  const creatorPanelShortcut = dom('span', getCssKeys(allCommands.creatorPanel.humanKeys));
+  const shortcutsModal = dom('span', getCssKeys(allCommands.shortcuts.humanKeys));
+  const accessibilityModal = dom('span', getCssKeys(allCommands.accessibility.humanKeys));
+  return cssSection(
+    cssModalSubheading(t("Keyboard navigation"), {role: "heading", "aria-level": 2}),
+    dom('p', t("On a document page, keyboard navigation is first locked on the current widget.")),
+    dom('p', t("Focus on other parts of the user interface using the following shortcuts:")),
+    dom('ul',
+      cssShortcutRow(t("{{nextRegionShortcut}} Focus on the next region", {nextRegionShortcut})),
+      cssShortcutRow(t("{{prevRegionShortcut}} Focus on the previous region", {prevRegionShortcut})),
+      cssShortcutRow(t("{{creatorPanelShortcut}} Focus to and from the creator panel", {creatorPanelShortcut})),
+    ),
+    dom('p', t("\"Regions\" are what we call the different parts of the user interface:")),
+    dom('ul',
+      dom('li', t("The left panel, home of the main navigation.")),
+      dom('li', t("The top panel, or the document header.")),
+      dom('li', inlineMarkdown(t(
+        "On document pages, each [widget]({{supportPageUrl}}) is a region that can receive focus.",
+        {supportPageUrl: commonUrls.helpWidgets}
+      ))),
+      dom('li', t("On non-document pages, the main content area is a region.")),
+      dom('li', t("Finally, the right panel – or the creator panel – is only available \
+through its own shortcut and is not included in the next and previous region cycle.")),
+    ),
+    cssModalSubheading(t("Other important keyboard shortcuts"), {role: "heading", "aria-level": 2}),
+    dom('ul',
+      cssShortcutRow(t("{{shortcutsModal}} Show the complete list of keyboard shortcuts", {shortcutsModal})),
+      cssShortcutRow(t("{{accessibilityModal}} Show the accessibility options (this modal)", {accessibilityModal})),
+    ),
+  );
+};
+
+const getCssKeys = (keys: string[]) => {
+  return keys.map((k, i) => i === keys.length - 1 ? cssKey(k) : [cssKey(k), t(" or ")]);
+};
+
+const cssSection = styled('div', `
+  margin-bottom: 32px;
+  &:last-child {
+    margin-bottom: 0;
+  }
+  ${nestedLinkStyles}
+`);
+
+const cssKey = styled('span', `
+  display: inline-block;
+  font-family: ${tokens.fontFamilyData};
+  color: ${tokens.body};
+  background-color: ${tokens.bgSecondary};
+  border: 1px solid ${tokens.decoration};
+  border-bottom: 3px solid ${tokens.decoration};
+  padding: 0.2em 0.4em;
+  border-radius: 0.2em;
+`);
+
+const cssShortcutRow = styled('li', `
+  display: flex;
+  align-items: center;
```

Comment:
I think `align-iterms: baseline` would make it look more aligned. Related, I noticed that the two shortcuts that appear in one line look misaligned and have different heights (I think because of the ⌘ symbol). Setting a suitable `line-height` on them should make them the same.